### DISCUSSION TITLE
Check if the form is edited or not ??

### DISCUSSION THREAD

**Discussion by udithv** [react-hook-form/react-hook-form#3203] [General]
I am using the same form for both updating and creating.. Is there a way to determine whether a form is edited or not. It would help as I don't need to waste another request to the server. 

Thank you,

**Comment by bluebill1049** [comment]
isDirty?

**Reply by udithv** [reply]
is there  a way I can access  the formState in the onSubmit function or am I better off binding it to the onSubmit function when i call handleSubmit function......

**Reply by bluebill1049** [reply]
```
const { formState: { isDirty }}  = useForm();

const onSubmit = () => {
  console.log(isDirty) 
}
``` 
?
